src/GoBoard.d.ts

```diff
@@ -0,0 +1,78 @@
+export type Vertex = [number, number]
+export enum Sign {
+    White = -1,
+    Empty = 0,
+    Black = 1,
+}
+export type SignMap = Sign[][]
+
+declare const alpha: string
+
+declare function vertexEquals([x1, y1]: Vertex, [x2, y2]: Vertex): boolean
+
+declare class GoBoard {
+    public signMap: SignMap
+    public height: number
+    public width: number
+
+    private _players: [Sign, Sign]
+    private _captures: [number, number]
+    private _koInfo: {sign: Sign; vertex: Vertex}
```

Comment:
These are implementation details, so please remove them as well.

Reply:
Removed in https://github.com/SabakiHQ/go-board/pull/3/commits/9ce2874c6bcce502e957e68a60abb71968a78172